app/models/concerns/exportable.rb

```diff
@@ -147,6 +153,27 @@ def get_field_value_for_serialization(field)
       value
     end
   end
+  
+  def external_pledge_count
+    external_pledges.count
+  end
+  
+  def external_pledge_sum
+    sum = 0
+    all_pledges = external_pledges.all
+    
+    all_pledges.each do |pledge|
```

Comment:
An FYI -- I'll merge this as is, but next time you run into this you can do this immutably with `reduce`/`inject`, which is good for doing this with larger datasets. 

Something like:

```
def external_pledge_sum
  all_pledges.reduce { |accumulator = 0, pledge| pledge.try(:amount) || nil }
end
```